features/step_definitions/cli_steps.ts

```diff
@@ -1,12 +1,12 @@
-import { When, Then, DataTable } from '../../'
+import { DataTable, Then, When } from '../../'
```

Comment:
Did you reorder these manually? If we want them ordered we should consider an eslint rule. Personally don't feel strongly about it though

Reply:
No - I’m not sure whether that was webstorm or prettier to be honest.